utilities/mng.js

```diff
@@ -0,0 +1,100 @@
+const path = require('path');
+const { MongoClient } = require('mongodb');
+const selectLanguage = require('./selectLanguage');
+const { URL, DBNAME } = require('./dbSettings');
+const textObj = require(path.join(__dirname, '../public/text.json'));
+const connObj = {
+  client: null,
+  db: null,
+  collections: null
+};
+
+/**
+ * connect to the mongodb database using the provided parameters
+ */
+async function connect() {
+  // if we are already connected 
+  // no need to reconnect
+  if (connObj.client && connObj.db) {
```

Comment:
You'll need to change the logic here, or explicitly check for a connection, because it's possible for connObj.client && connObj.db == true even when no connection exists.

Checkout this api from the NodeJs driver we're using: http://mongodb.github.io/node-mongodb-native/3.1/api/MongoClient.html#isConnected

Reply:
Yes that would be better. I will try to fix this!